src/main/java/entity/Attachment.java

```diff
@@ -0,0 +1,82 @@
+package entity;
+
+import org.springframework.web.multipart.MultipartFile;
+
+import org.springframework.cloud.gcp.data.spanner.core.mapping.PrimaryKey;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.Column;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.Table;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.NotMapped;
+
+@Table(name = "Attachment")
+public final class Attachment {
+    
+    @PrimaryKey
+    @Column(name = "AttachmentID")
+    private long attachmentId;
+
+    @Column(name = "FileName")
+    private String fileName;
```

Comment:
Please add some documentation about this field.
Things which are unclear - does it include the extension of the file?

Reply:
Done